### ISSUE TITLE
Feature request: display car name instead of speed (or give an option to do one of them)

### ISSUE THREAD

**Issue by sidtr0** [jaiden-d/FH5RP#1]

**Comment by jaaiden** [comment]
I thought I added this at some point from FH4RP, whoops! I'll look into adding this soon! 😄

**Comment by silentcontribution** [comment]
Looking forward to that!

**Comment by sidtr0** [comment]
@jaiden-d I just forked the code and took a look at `TelemetryData.cs`; in FH4RP you used https://forzadb.dgtl.dev/ but it doesn't seem to be updated for FH5 :/

**Comment by jaaiden** [comment]
Yeah when I setup that website it was mainly meant specifically for FH4, and I noticed in my testing that some of the car ordinal IDs (the unique IDs that each car has) sometimes don't match between FH4 and FH5. Because of this, I need to figure out a different storage solution for the car information instead.

**Comment by primeeeyk** [comment]
@jaiden-d i tried to use the rpc, but it's not working at all. can you help a man?
